source/2020-07-31-the-ember-times-issue-158.md

```diff
@@ -100,14 +101,15 @@ Read more on the [blog post on dev.to](https://dev.to/_raja_sk_/demystifying-emb
 
 ---
 
-## [Section title in sentence case 🐹](section-url)
+## [Ember 3.20 Released 🚀](https://blog.emberjs.com/2020/07/29/ember-3-20-released.html)
 
-<change section title emoji>
-<consider adding some bold to your paragraph>
-<please include link to external article/repo/etc in paragraph / body text, not just header title above>
+There's a new [Ember.js blog post](https://blog.emberjs.com/2020/07/29/ember-3-20-released.html) up that announces the release of version 3.20 of Ember.js, Ember Data and Ember CLI! 
 
-<add your name to author list, top and bottom>
-<add blurb and emoji to "SOME-INTRO-HERE">
+3.20 includes several new features for the ecosystem. In Ember.js there's an `in-element` helper now available as public API. This helper solves rendering challenges similar to `ember-wormhole` or `ember-elsewhere`, and allows rendering content into a destination elsewhere on a page. Ember Data now explicitly supports the use of `EmbeddedRecordsMixin` and `JSONAPISerializer` using the `isEmbeddedRecordsMixinCompatible` property in your serializers. Ember CLI now allows syncing Blueprints when running `npx ember-cli-update`, avoiding some potential issues that previously existed when running this flow - checkout the [related RFC](https://emberjs.github.io/rfcs/0477-blueprints-update.html) to learn more.
```

Comment:
I think I'd recommend making a list to describe the features. I also shortened some phrases below. Can you take a look?

```suggestion
3.20 includes several new features for the ecosystem.

- In Ember.js, the `{{in-element}}` helper is available as public API. This helper solves rendering challenges similarly to `ember-wormhole` and `ember-elsewhere`. It allows rendering content into a destination elsewhere on a page.
- Ember Data explicitly supports the combined use of `EmbeddedRecordsMixin` and `JSONAPISerializer` through the `isEmbeddedRecordsMixinCompatible` property.
- Ember CLI allows syncing Blueprints when running `npx ember-cli-update`, which avoids some potential issues that previously existed when running this flow. Check out the [related RFC](https://emberjs.github.io/rfcs/0477-blueprints-update.html) to learn more.
```